testing/tests/ui/let_destructuring_has_rest.rs

```diff
@@ -0,0 +1,32 @@
+use rinja::Template;
+
+struct X {
+    a: u32,
+    b: u32,
+}
+
+#[derive(Template)]
+#[template(source = "
+{%- if let X { a, .., } = x -%}hello {{ a }}{%- endif -%}
+", ext = "html")]
+struct T1 {
+    x: X,
+}
+
+#[derive(Template)]
+#[template(source = "
+{%- if let X { a .. } = x -%}hello {{ a }}{%- endif -%}
+", ext = "html")]
+struct T2 {
+    x: X,
+}
+
+#[derive(Template)]
+#[template(source = "
+{%- if let X { a, 1 } = x -%}hello {{ a }}{%- endif -%}
+", ext = "html")]
+struct T3 {
```

Comment:
Just thought about another case:

```
let X { a, .., b } = x
```

and also:

```
let X { .., a } = x
```

I suppose the parsing will fail but better ensure the errors are good.

Reply:
I expanded the error message a bit to give better information how to use the `..` in named structs.